cli/src/pcluster/templates/login_nodes_stack.py

```diff
@@ -83,6 +93,81 @@ def _add_login_nodes_pool_launch_template(self):
                 metadata_options=ec2.CfnLaunchTemplate.MetadataOptionsProperty(
                     http_tokens=get_http_tokens_setting(self._config.imds.imds_support)
                 ),
+                user_data=Fn.base64(
+                    Fn.sub(
+                        get_user_data_content("../resources/login_node/user_data.sh"),
+                        {
+                            **{
+                                "RAIDSharedDir": to_comma_separated_string(
+                                    self._shared_storage_mount_dirs[SharedStorageType.RAID]
+                                ),
+                                "RAIDType": to_comma_separated_string(
+                                    self._shared_storage_attributes[SharedStorageType.RAID]["Type"]
+                                ),
```

Comment:
I'm not sure this is needed.

Reply:
Let's open a follow-up task to clean up the dna.json for compute and login nodes. I'm not sure we need everything listed here.